tests/summarywriter_test.cpp

```diff
@@ -107,6 +107,29 @@ TEST_CASE_METHOD(SummaryWriterTestsFixture, "Writing text summary to file",
       " Project developed during GSoc 2020 ", *f1);
 }
 
+/**
+ * Test embedding support.
+ */
+TEST_CASE_METHOD(SummaryWriterTestsFixture,
+                 "Writing embedding summary to file", "[SummaryWriter]")
+{	
+  arma::mat temp;
+  mlpack::data::Load("./data/vecs.tsv", temp);
+  std::vector<std::string> meta;
+  std::string line;
+  std::ifstream meta_file("./data//meta.tsv");
+  while (getline(meta_file, line))
+  {
+    meta.push_back(line);
+  }
+  meta_file.close();
+
+  // Make sure you pass the correct dimesnions here
```

Comment:
```suggestion
  // Make sure you pass the correct dimensions here.
```

I don't think we need this line as we are not passing any matrix neither any dimensions.